llvm/unittests/CodeGen/LowLevelTypeTest.cpp

```diff
@@ -300,6 +321,7 @@ TEST(LowLevelTypeTest, Invalid) {
   ASSERT_FALSE(Ty.isScalar());
   ASSERT_FALSE(Ty.isPointer());
   ASSERT_FALSE(Ty.isVector());
+  ASSERT_FALSE(Ty.isToken());
```

Comment:
This test should be separately fixed to consistently use EXPECT_*

Reply:
Sounds good.